package.json

```diff
@@ -1,4 +1,5 @@
 {
+{
```

Comment:
@col1985  Looks like some extra braces got in here and at the bottom of the file.
I'd fix it, but i'm not sure I have push access to your fork. I'll try